test/front-matter-ignore-line.md

```diff
@@ -0,0 +1,9 @@
+---
+a: b
+c: d
+e: f
+g: h
+---
+
+<!-- markdownlint-disable-next-line -->
+<iframe width="100%" height="166" scrolling="no" frameborder="no" allow="autoplay" src="http:www.google.com/very+long+url"></iframe>
```

Comment:
This could be any HTML and the suppression just for the "inline-html" rule, I think?

Reply:
Specifically, please use a simpler example like "<hr>".